build.gradle

```diff
@@ -9,6 +17,9 @@ apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply plugin: 'antlr'
 
+apply plugin: 'osgi'
```

Comment:
@tdraier Thank you. Could you add a short comment here what both plugins actually do?